examples/rag-wikipedia-ollama/main.go

```diff
@@ -56,7 +56,7 @@ func main() {
 	}
 	// Add docs to the collection, if the collection was just created (and not
 	// loaded from persistent storage).
-	docs := []chromem.Document{}
+	var docs []chromem.Document
```

Comment:
For this one, I like to differentiate between cases where it's certain that the slice is going to be filled with elements (e.g. this code example should have 200 elements, otherwise the example is broken), vs cases where it's not certain and where you might later want to check for `docs == nil` (e.g. in production code with user input).

But that might just be me, and not a big deal, so no blocker